src/cli_util.rs

```diff
@@ -1386,15 +1386,25 @@ fn load_revset_aliases(
 pub fn resolve_base_revs(
     workspace_command: &WorkspaceCommandHelper,
     revisions: &[RevisionArg],
+    multi: bool,
```

Comment:
I just changed it to `--allow-large-revsets`. WDYT?

Reply:
That makes me think it's about allowing revsets over 100 commits or something instead of erroring out with "The revset resolves to > 100 commits. Did you really mean that?". We don't have an error like that, but users might think we do when they see the flag name. It still seems fine to me since it's temporary.

Reply:
Is `allow-plural-revsets` better or more confusing? It feels a bit weird to type, and I guess then the short option would have to be `-P`.

Reply:
Sorry about suggesting one more thing, but given revision (or "singular revset") is the default argument already, just `--allow-revsets` is arguably clear (and concise) enough?

Reply:
It's an idea. It's hard for me to judge whether that's confusing or not. Strictly speaking, if `A` has a single parent, `A-` is a revset even though you don't need `--allow-large-revsets` to do `-d A-` in that case.

Reply:
I agree that `--allow-revsets` is misleading because even `A` is a revset, even if it resolves to a single revision. My comment was really just about the argument name in this method, which I didn't think needed to match the name on the CLI. But let's not block this PR on the name of either of them, especially since we're hoping to enable it by default soon.

Reply:
I renamed the argument, but I'm thinking of sticking with `--allow-large-revsets` for now.